src/test/java/picard/IntelInflaterDeflaterLoadTest.java

```diff
@@ -0,0 +1,37 @@
+package picard;
```

Comment:
Here's a test for the successful loading of the Intel inflater / deflater, modeled off of the [GATK tests](https://github.com/broadinstitute/gatk/blob/master/src/test/java/org/broadinstitute/hellbender/IntelInflaterDeflaterIntegrationTest.java).